setup.py

```diff
@@ -0,0 +1,17 @@
+"""Creating a package."""
+
+from setuptools import setup
+
+dependencies = ['ipython', 'pytest', 'pytest-watch', 'tox']
+
+setup(
+    name='Queue',
```

Comment:
Your package isn't just for the "Queue" data structure, so you should probably call it something more relevant to a repository holding multiple data structs